mz_bokeh_package/utilities/auth.py

```diff
@@ -2,58 +2,43 @@
 This module takes care of the authentication of calls to bokeh apps
 
 """
-
-import requests
 import os
-from bokeh.io import curdoc
 from tornado.web import RequestHandler
-from typing import Optional
 
 from mz_bokeh_package.utilities.environment import Environment
+from mz_bokeh_package.utilities.graphql_api import GraphqlQueryError
+from mz_bokeh_package.utilities.current_user import CurrentUser
 
 
-def get_user(request_handler: RequestHandler) -> str:
+def get_user(request_handler: RequestHandler) -> bool | None:
     """
-    authenticate user based on api_key and user_key that are sent via the query parameters of the request and
+    authenticate user based on api_key that are sent via the query parameters of the request and
     return the user_id if the user is authenticated and otherwise return None
 
     Args:
         request_handler: a tornado RequestHandler object that contains the query parameters of the request,
-            which contains the api_key and user_key of the user
+            which contains the api_key of the user
 
     Returns:
-        the user id when authentication is successful, and otherwise None
+        True when authentication is successful, and otherwise None
     """
 
     # bypass authentication in the case of health.py dashboard to allow GCP to perform health checks
     if request_handler.request.path.split("/")[-1] in ("health", "error"):
-        return "ok"
-
-    query_arguments = request_handler.request.query_arguments
+        return True
 
-    # get the api_key from the request header
-    api_keys = query_arguments.get("api_key")
-    if api_keys is not None and len(api_keys) == 1:
-        api_key = api_keys[0]
-    else:
-        api_key = ""
-
-    # get the user_key from the request header
-    user_keys = query_arguments.get("user_key")
-    if user_keys is not None and len(user_keys) == 1:
-        user_key = user_keys[0]
-    else:
-        user_key = ""
-
-    # in the development environment, allow overriding the api_key and user_key via env variables
+    # in the development environment, allow overriding the api_key via env variables
     if Environment.get_environment() == 'dev':
```

Comment:
I think it's better to use `CurrentUser.get_api_key()` here to avoid duplication of logic. You just need to make sure that it doesn't crash when there is no session (and if it does, you should refactor it to not crash).